### DISCUSSION TITLE
new feature: support  Vue CLI 5 can load both `vue.config.js` and `vue.config.ts`

### DISCUSSION THREAD

**Discussion by Viggo95** [vuejs/vue-cli#6773] [Ideas]
I want to implement a pr that supports `@vue/cli-service@5`  to load both `vue.config.js` and `vue.config.ts`

Does anyone have an opposing or agreeing opinion?  The effect will look like this 👇

<img src="https://user-images.githubusercontent.com/22092110/138402621-0ae1ae59-0c98-41e9-9009-ac813d00da1f.png" height="300px">

I expect to finish it in  this week

<br>

The advantage of this feature is that the configuration file in the TS project is also `vue.config.ts`
The DX will be better

<br>

@sodatea    @fangbinwei

**Comment by haoqunjiang** [comment]
Cool!
